packages/accordions/stories/4-Timeline.stories.tsx

```diff
@@ -15,26 +15,24 @@ import EmailIcon from '@zendeskgarden/svg-icons/src/12/email-stroke.svg';
 import CartIcon from '@zendeskgarden/svg-icons/src/12/shopping-cart-stroke.svg';
 import ClipboardIcon from '@zendeskgarden/svg-icons/src/12/clipboard-blank-stroke.svg';
 
-const iconSize = { width: 11, height: 11 };
```

Comment:
☝️ Direct attribute manipulation like this in storybook is a solid clue that the underlying component should be enhanced to cover expected cases.